pytest.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
 env =
     AWS_DEFAULT_REGION=eu-west-2
-    
+addopts = --tb=short
```

Comment:
I'm thinking that this would be better off in your IDE's configuration (or in your global pytest.ini) rather than enforcing for all users, as it could be useful in some cases.

Reply:
As discussed outside of here, less noise as the default in the repository ones is best and we can just delete the line if/when we think the extra output might help.

It links you to the file anyway so we can just click through if we want file contents.